provision/virtualbox.sh

```diff
@@ -3,5 +3,5 @@
 set -e
 set -x
 
-sudo pacman -S --noconfirm virtualbox-guest-utils-nox virtualbox-guest-modules-arch
+sudo pacman -S --noconfirm virtualbox-guest-utils-nox
```

Comment:
why did you remove virtualbox-guest-modules-arch? it has nothing to do with the issue you've mentioned

Reply:
virtualbox-guest-modules-arch is not available in the archlinux repos, and the build failed with a "no target found" on pacman (however it is part of linux 5.6, that is obviously installed). So it is part of the "Fix virtualbox build". With this PR I was only fixing the build for virtualbox, no functional update. I will do another PR that fixes reflector-init service by waiting for network-online (instead of network) and before sshd is available, so if somebody uses the box with a pacman install at the begining of his Vagrantfile he has the right mirrorlist.